shared/metadata.go

```diff
@@ -52,14 +57,40 @@ func (m MetadataResults) Len() int           { return len(m) }
 func (m MetadataResults) Swap(i, j int)      { m[i], m[j] = m[j], m[i] }
 func (m MetadataResults) Less(i, j int) bool { return m[i].Test < m[j].Test }
 
+// GetMetadatURL returns the URL that fetches Metadata information.
+// For testing, it returns a MetadataTestingURL set by users.
+func GetMetadatURL(isTesting bool) string {
```

Comment:
What is this for? I don't see any `GetMetadatURL(true)` anywhere? The empty "testing" URL never seems to be used.

Also, it's `Metadata` not `Metadat`.